### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by benjello**
Thanks for contributing to OpenFisca! Remove this line, as well as any other in the following that don't fit your contribution  :)

#### Technical changes

- Fix average_rate computation when division by zero are present
- Fix entity.count restoration for a simulation dump

### CODE REVIEW

File: openfisca_core/rates.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 def average_rate(target = None, varying = None):
     # target: numerator, varying: denominator
-    return 1 - target / (varying * (varying != 0) + (varying == 0))
+    return 1 - target / varying
```
Comment (Morendil) [inline openfisca_core/rates.py]: Where is average_rate used ? I can't find references to this function.

Comment (benjello) [reply openfisca_core/rates.py]: It is used in many packages like openfisca-matplotlib, notebooks etc. It is so generic that it is better to have it include in core. It is not worth to maintenance cost to have a specific package for that.

Comment (Morendil) [reply openfisca_core/rates.py]: If it's worth having in Core, it's probably worth having a unit test for it. No unit tests and no obvious callers means zero pressure on maintainers to keep it operational. :)

Comment (maukoquiroga) [reply openfisca_core/rates.py]: I can add it, but I do not know what it does, example:

```python
import numpy

from openfisca_core.rates import average_rate


def test_average_rate_when_varying_is_zero():
    '''Add description like: it should do this'''
    target = numpy.array([123])
    result = average_rate(target, varying = 0)
    assert numpy.isinf(result)


def test_average_rate_when_varying_is_not_zero():
    '''Add description like: it should do that'''
    target = numpy.array([1, 2, 3])
    result = average_rate(target, varying = 2)
    expected = numpy.array([.5, 0, -.5])
    numpy.testing.assert_equal(result, expected)
```

Comment (maukoquiroga) [reply openfisca_core/rates.py]: @Morendil see #760

**Pull request merged**